### ISSUE TITLE
Funcion INVERT y ORDER

### ISSUE THREAD

**Issue by JaviL13** [IIC2133-PUC/2023-2#85]
En el enunciado mencionan que la función invert no puede afectar a DEEP-ORDER, pero puede afectar a la ORDER. 
Gracias

**Comment by CataAlamos** [comment]
Hola! No entiendo bien tu duda, me lo puedes explicar otra vez?

**Comment by JaviL13** [comment]
si claro, si tenemos el arbol 
   2
1    3
y hacemos order nos queda 1 2 3 
si hacemos invert 
   2
3    1
mi duda es si despues de invert volvemos a hacer Order queda 
a) 1 2 3 
b) 3 2 1

**Comment by CataAlamos** [comment]
Hola! Si vez el enunciado un árbol "normal" debería verse así
<img width="283" alt="Screenshot 2023-09-09 at 17 11 29" src="https://github.com/IIC2133-PUC/2023-2/assets/62016215/eedfbcbf-2829-4175-9790-da7775e574af">
y si le haces order debería darte 2 3 4 5 6 7 10
En caso de que el árbol este invertido se ve así
<img width="290" alt="Screenshot 2023-09-09 at 17 12 44" src="https://github.com/IIC2133-PUC/2023-2/assets/62016215/9b007270-b9b5-4383-90ac-5f426ddfa2f2">
En caso de que este invertido el orden debes leerlo igual, porque quieres que se entreguen los datos ordenados; lo que cambia es como lees los datos en orden.

**Comment by JaviL13** [comment]
Osea, si hago order al segundo arbol deberia entregar 
3 6 10 7 4 5 2 
??

**Comment by CataAlamos** [comment]
No, deberia ser el mismo orden! Porque orden implica que este de menor a mayor
